src/stream_processor/parser/sql.y

```diff
@@ -84,7 +84,7 @@ void yyerror(struct flb_sp_cmd *cmd, const char *query, void *scanner, const cha
 %type <string>     record_key
 %type <string>     prop_key
 %type <string>     prop_val
-%type <expression> condition
+%type <expression> condition condition1 condition2
```

Comment:
Better to have more clear naming. For instance, `condition and_condition or_condition`.